src/main/java/org/subethamail/smtp/server/Session.java

```diff
@@ -405,13 +417,14 @@ public void setDeclaredMessageSize(int declaredMessageSize) {
 
     /**
      * Starts a mail transaction by creating a new message handler.
-     * 
+     *
      * @throws IllegalStateException
      *             if a mail transaction is already in progress
      */
     public void startMailTransaction() throws IllegalStateException {
-        if (this.messageHandler != null)
+        if (this.messageHandler != null) {
```

Comment:
Sorry I didn't revert this change, it add implicit {} aroung one line statement.
I can revert this if unwanted

Reply:
no problem, we'll leave